Index: src/main/java/p2parking/dao/UsuariosDAO.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package p2parking.dao;\n\nimport p2parking.jdo.Usuario;\n\nimport java.util.List;\n\npublic class UsuariosDAO extends DataAccessObjectBase implements iAccesoObjeto<Usuario> {\n    @Override\n    public void save(Usuario object) {\n\n    }\n\n    @Override\n    public void delete(Usuario object) {\n\n    }\n\n    @Override\n    public List<Usuario> getAll() {\n        return null;\n    }\n\n    @Override\n    public Usuario find(String param) {\n        return null;\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/p2parking/dao/UsuariosDAO.java b/src/main/java/p2parking/dao/UsuariosDAO.java
--- a/src/main/java/p2parking/dao/UsuariosDAO.java	(revision 9a054af959e41f6c01654d0db769981413f7a1da)
+++ b/src/main/java/p2parking/dao/UsuariosDAO.java	(date 1648487810890)
@@ -1,6 +1,6 @@
 package p2parking.dao;
 
-import p2parking.jdo.Usuario;
+import p2parking.serialization.Usuario;
 
 import java.util.List;
 
